Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM maven:3.8.7-eclipse-temurin-17 AS build
+COPY . /home/app
+WORKDIR /home/app
+RUN mvn -B -U clean package -pl bpdm-certificate-management -am -DskipTests
+
+FROM eclipse-temurin:17-jre-alpine
+# Adding wget for the health check
+RUN apk --no-cache add wget
+COPY --from=build /home/app/bpdm-certificate-management/target/bpdm-certificate-management.jar /usr/local/lib/bpdmCertificate/app.jar
+RUN apk update && apk upgrade --no-cache libssl3 libcrypto3
+ARG USERNAME=bpdmCertificate
+ARG USERID=10001
+ARG GID=10002
+RUN addgroup -g $GID -S $USERNAME
+RUN adduser -u $USERID -S $USERNAME $USERNAME
+USER $USERNAME
+WORKDIR /usr/local/lib/bpdmCertificate
+EXPOSE 8080
```

Comment:
Here it would make sense to expose 8086 as the application runs on port 8086 on default.

Better yet to provide it as a build argument with default 8086 and then used in the expose and the healthcheck.

Yes, this is also wrong in BPDM at the moment

Reply:
exposed to 8086